src/main/java/rayshawn/chatbot/messages/Messages.java

```diff
@@ -0,0 +1,17 @@
+package rayshawn.chatbot.messages;
+
+/**
+ * Container for user visible messages
+ */
+public class Messages {
```

Comment:
Good practice on using constants for values that will be used repeatedly.

Reply:
I agree! I did not even think of this!

Reply:
I agree! I did not even think of this!